### DISCUSSION TITLE
How do I disable these 'H' in outline?

### DISCUSSION THREAD

**Discussion by CrocodileDuty** [SlRvb/Obsidian--ITS-Theme#249] [Q&A]
![image](https://github.com/SlRvb/Obsidian--ITS-Theme/assets/32086892/285226d1-797e-497d-b65c-798ba6ad0c27)

**Comment by SlRvb** [comment]
Oops, seems like I hadn't transferred this feature over, could you update to ITS v1.1.34 and toggle on the `Hide Outline Plugin Heading Icons` under `Workspace > Minimalist Workspace` in the style settings?
